### PR TITLE
VBA Change Call-First to Online Tools and Customer Service Number on Detail page 

### PR DESCRIPTION
**Pull request by brangi**
## Description
Issue: https://github.com/department-of-veterans-affairs/va.gov-team/issues/9496

## Testing done
Locally & http://7740b9f2467d6789ab9eb17e95ce6c9a.review.vetsgov-internal/find-locations/facility/vba_349n

## Screenshots
Screen Shot 2020-07-15 at 12 33 47 PM

## Acceptance criteria
- [x] Add link to VBA online customer service tool, phone number or other benefit website.

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/applications/facility-locator/components/VABenefitsCall.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+export default function VABenefitsCall() {
+  return (
+    <div>
+      <p className="p1">
+        <strong>To check on the status of your claim or appeal</strong>, use our{' '}
+        <a href="/claim-or-appeal-status">online claim status tool</a>.
+      </p>
+      <p className="p1">
+        <strong>To get help with benefits</strong>, call{' '}
```
Comment (mpelzsherman) [inline src/applications/facility-locator/components/VABenefitsCall.jsx:17]: Would it be better to use `&nbsp;` instead of `{' '}`?

Comment (brangi) [reply src/applications/facility-locator/components/VABenefitsCall.jsx:17]: Well, `{' '}` is added  by esli
Screen Shot 2020-07-15 at 2 32 17 PM
nt and is widely used in other components

File: src/applications/facility-locator/components/VABenefitsCall.jsx [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        <Telephone contact={contact} extension={extension}>
+          {' '}
+          {formattedPhoneNumber}
+        </Telephone>{' '}
```
Comment (mpelzsherman) [inline src/applications/facility-locator/components/VABenefitsCall.jsx:21]: Please note that the Telephone component will throw an Error if the contact is < 10 characters long.

I have filed an [issue](contact) to have that removed, but in the meantime, you may want to do some error checking, or add an [ErrorBoundary](https://reactjs.org/docs/error-boundaries.html) component.

Comment (mpelzsherman) [inline src/applications/facility-locator/components/VABenefitsCall.jsx:21]: You can just use the constant here:

```js
        <Telephone contact={CONTACTS.VA_BENEFITS} />
```

Comment (brangi) [reply src/applications/facility-locator/components/VABenefitsCall.jsx:21]: Tried this but fails 
Screen Shot 2020-07-15 at 4 31 12 PM

Comment (brangi) [reply src/applications/facility-locator/components/VABenefitsCall.jsx:21]: never-mind, that was my bad, I was using the import incorrectly

**Review by mpelzsherman — APPROVED** [review]
Nice job on the unit tests!

**Pull request merged**